### ISSUE TITLE
ActionView::Template::Error: Missing partial devise/registrations/_alert, devise/_alert, application/_alert with {:locale=>[:en], :formats=>[:html], :variants=>[], :handlers=>[:raw, :erb, :html, :builder, :ruby, :md, :markdown, :slim, :haml]}. Searched in

### ISSUE THREAD

**Issue by stacietaylorcima** [coyote-team/coyote#416]
View details in Rollbar: [https://rollbar.com/livecoyote/livecoyote/items/46/](https://rollbar.com/livecoyote/livecoyote/items/46/)

```

ActionView::MissingTemplate: Missing template devise/registrations/update, devise/update, application/update with {:locale=>[:en], :formats=>[:html], :variants=>[], :handlers=>[:raw, :erb, :html, :builder, :ruby, :md, :markdown, :slim, :haml]}. Searched in:
  * "/app/app/views"
  * "/app/lib/scavenger_hunt/app/views"
  * "/app/vendor/bundle/ruby/2.4.0/gems/kaminari-core-1.1.1/app/views"
  * "/app/vendor/bundle/ruby/2.4.0/gems/great_pretender-0.1.5/app/views"
  * "/app/vendor/bundle/ruby/2.4.0/bundler/gems/easymarklet-53829a6fbd80/app/views"
  * "/app/vendor/bundle/ruby/2.4.0/gems/devise-4.5.0/app/views"
  * "/app/vendor/bundle/ruby/2.4.0/gems/apipie-rails-0.5.10/app/views"
  * "/app/vendor/bundle/ruby/2.4.0/gems/analytical-3.0.12/app/views"

  File "/app/vendor/bundle/ruby/2.4.0/gems/actionview-5.2.0/lib/action_view/path_set.rb", line 48, in find
  File "/app/vendor/bundle/ruby/2.4.0/gems/actionview-5.2.0/lib/action_view/lookup_context.rb", line 116, in find
  File "/app/vendor/bundle/ruby/2.4.0/gems/actionview-5.2.0/lib/action_view/renderer/abstract_renderer.rb", line 20, in find_template
  File "/app/vendor/bundle/ruby/2.4.0/gems/actionview-5.2.0/lib/action_view/renderer/template_renderer.rb", line 40, in determine_template
  File "/app/vendor/bundle/ruby/2.4.0/gems/actionview-5.2.0/lib/action_view/renderer/template_renderer.rb", line 10, in render
  File "/app/vendor/bundle/ruby/2.4.0/gems/actionview-5.2.0/lib/action_view/renderer/renderer.rb", line 44, in render_template
  File "/app/vendor/bundle/ruby/2.4.0/gems/actionview-5.2.0/lib/action_view/renderer/renderer.rb", line 25, in render
  File "/app/vendor/bundle/ruby/2.4.0/gems/actionview-5.2.0/lib/action_view/rendering.rb", line 103, in _render_template
  File "/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.2.0/lib/action_controller/metal/streaming.rb", line 219, in _render_template
  File "/app/vendor/bundle/ruby/2.4.0/gems/actionview-5.2.0/lib/action_view/rendering.rb", line 84, in render_to_body
  File "/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.2.0/lib/action_controller/metal/rendering.rb", line 52, in render_to_body
  File "/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.2.0/lib/action_controller/metal/renderers.rb", line 142, in render_to_body
  File "/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.2.0/lib/abstract_controller/rendering.rb", line 25, in render
  File "/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.2.0/lib/action_controller/metal/rendering.rb", line 36, in render
  File "/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.2.0/lib/action_controller/metal/instrumentation.rb", line 46, in block (2 levels) in render
  File "/app/vendor/bundle/ruby/2.4.0/gems/activesupport-5.2.0/lib/active_support/core_ext/benchmark.rb", line 14, in block in ms
  File "/app/vendor/ruby-2.4.1/lib/ruby/2.4.0/benchmark.rb", line 308, in realtime
  File "/app/vendor/bundle/ruby/2.4.0/gems/activesupport-5.2.0/lib/active_support/core_ext/benchmark.rb", line 14, in ms
  File "/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.2.0/lib/action_controller/metal/instrumentation.rb", line 46, in block in render
  File "/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.2.0/lib/action_controller/metal/instrumentation.rb", line 87, in cleanup_view_runtime
  File "/app/vendor/bundle/ruby/2.4.0/gems/activerecord-5.2.0/lib/active_record/railties/controller_runtime.rb", line 31, in cleanup_view_runtime
  File "/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.2.0/lib/action_controller/metal/instrumentation.rb", line 45, in render
  File "/app/vendor/bundle/ruby/2.4.0/gems/responders-2.4.0/lib/action_controller/responder.rb", line 238, in default_render
  File "/app/vendor/bundle/ruby/2.4.0/gems/responders-2.4.0/lib/action_controller/responder.rb", line 170, in to_html
  File "/app/vendor/bundle/ruby/2.4.0/gems/responders-2.4.0/lib/action_controller/responder.rb", line 163, in respond
  File "/app/vendor/bundle/ruby/2.4.0/gems/responders-2.4.0/lib/action_controller/responder.rb", line 156, in call
  File "/app/vendor/bundle/ruby/2.4.0/gems/responders-2.4.0/lib/action_controller/respond_with.rb", line 211, in respond_with
  File "/app/vendor/bundle/ruby/2.4.0/gems/devise-4.5.0/app/controllers/devise/registrations_controller.rb", line 63, in update
  File "/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.2.0/lib/action_controller/metal/basic_implicit_render.rb", line 6, in send_action
  File "/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.2.0/lib/abstract_controller/base.rb", line 194, in process_action
  File "/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.2.0/lib/action_controller/metal/rendering.rb", line 30, in process_action
  File "/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.2.0/lib/abstract_controller/callbacks.rb", line 42, in block in process_action
  File "/app/vendor/bundle/ruby/2.4.0/gems/activesupport-5.2.0/lib/active_support/callbacks.rb", line 109, in block in run_callbacks
  File "/app/vendor/bundle/ruby/2.4.0/gems/audited-4.8.0/lib/audited/sweeper.rb", line 14, in around
  File "/app/vendor/bundle/ruby/2.4.0/gems/activesupport-5.2.0/lib/active_support/callbacks.rb", line 118, in block in run_callbacks
  File "/app/vendor/bundle/ruby/2.4.0/gems/audited-4.8.0/lib/audited/sweeper.rb", line 14, in around
  File "/app/vendor/bundle/ruby/2.4.0/gems/activesupport-5.2.0/lib/active_support/callbacks.rb", line 118, in block in run_callbacks
  File "/app/vendor/bundle/ruby/2.4.0/gems/activesupport-5.2.0/lib/active_support/callbacks.rb", line 136, in run_callbacks
  File "/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.2.0/lib/abstract_controller/callbacks.rb", line 41, in process_action
  File "/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.2.0/lib/action_controller/metal/rescue.rb", line 22, in process_action
  File "/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.2.0/lib/action_controller/metal/instrumentation.rb", line 34, in block in process_action
  File "/app/vendor/bundle/ruby/2.4.0/gems/activesupport-5.2.0/lib/active_support/notifications.rb", line 168, in block in instrument
  File "/app/vendor/bundle/ruby/2.4.0/gems/activesupport-5.2.0/lib/active_support/notifications/instrumenter.rb", line 23, in instrument
  File "/app/vendor/bundle/ruby/2.4.0/gems/activesupport-5.2.0/lib/active_support/notifications.rb", line 168, in instrument
  File "/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.2.0/lib/action_controller/metal/instrumentation.rb", line 32, in process_action
  File "/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.2.0/lib/action_controller/metal/params_wrapper.rb", line 256, in process_action
  File "/app/vendor/bundle/ruby/2.4.0/gems/activerecord-5.2.0/lib/active_record/railties/controller_runtime.rb", line 24, in process_action
  File "/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.2.0/lib/abstract_controller/base.rb", line 134, in process
  File "/app/vendor/bundle/ruby/2.4

**Comment by flipsasser** [comment]
Fixed w/#484

**Comment by flipsasser** [comment]
Weird; it didn't mark as close
